public/components/content-list-drawer/content-list-drawer.html

```diff
@@ -131,7 +131,7 @@
                             </div>
                             <div class="drawer__item-wordcount">
                                 <span class="drawer__item-title">Print word count</span>
-                                <span class="drawer__item-content">{{ contentItem.printWordCount }} </span>
+                                <span class="drawer__item-content">{{ contentItem.printWordCount || "Unknown" }} </span>
```

Comment:
@SHession this should fix what we were discussing earlier :)

Reply:
It might be worth adding that same bit to the line 130, just incase wordCount ever comes back as null

Reply:
I believe this should be handled already. At least, it is in my local.

![Screenshot 2020-05-13 at 15 30 12](https://user-images.githubusercontent.com/12645938/81825514-a8b9bb00-952e-11ea-8457-3b4ea5b2b339.png)

I think this value is pulled, along with some other "Unknown"s for headline, standfirst, etc, from the [`emptyCapiContentObject`](https://github.com/guardian/workflow-frontend/blob/c2f6a548877b30f5059bfce037d14ccb9c32ae3b/public/lib/capi-content-service.js#L71).

Reply:
Oh yeah, you're right. In which case it might be worth changing _content-list-item.js :91_ instead. That's a bit pedantic though, so if you don't want to change it that's fine.